docs/kintsugi/crowdloans.md

```diff
@@ -5,18 +5,26 @@ To participate in the crowdloan, you need to lock your KSM and vote for Kintsugi
 
 Below is a summary of the Kintsugi crowdloan details, as described in the [token economy whitepaper released by Kintsugi Labs](https://raw.githubusercontent.com/interlay/whitepapers/master/Kintsugi_Token_Economy.pdf).
 
-## Hard Facts
+
+###  Hard Facts
```

Comment:
Can you please make this a h2 (##)? If we want to control the size of h2, it should be via css but not changing the ordering on the headings.